ow_plexil/src/plans/FaultHandlingPattern2.plp

```diff
@@ -0,0 +1,51 @@
+// The Notices and Disclaimers for Ocean Worlds Autonomy Testbed for Exploration
+// Research and Simulation can be found in README.md in the root directory of
+// this repository.
+
+// Action 1: Pan antenna in 15 degree increments.
+// Action 2: Unstow and stow the arm repeatedly.
+// If no faults are occurring, action 1 and 2 occur simultaneously.
+// If an antenna fault but no arm faults occurring, only action 2 occurs.
+// If an arm fault but no antenna faults occurring, only action 1 occurs.
+// If both antenna and arm faults occurring, the plan waits for resolution.
+
+
+#include "plan-interface.h"
+
+LibraryAction InitializeAntennaAndArm ();
+
+FaultHandlingPattern2:
+{
+  Real NewAngle = 0;
+
+  LibraryCall InitializeAntennaAndArm ();
```

Comment:
This should not run unless there are no antenna or arm faults.   At present, if you start this plan with faults injected, this plan will run erroneously.  It could simply wait until there are no faults before starting.   This applies to patterns 1 and 3 as well.